src/components/AssetInfo/TokenSideBarContent.tsx

```diff
@@ -13,10 +13,10 @@ import {
 } from "../../services/token";
 import TokenTransactionSuccess from "./TokenTransactionSuccess";
 import { TOKEN_ACTION_TYPES, getSuccessResponse } from "./TokenActionUtil";
-const { trace, error } = getLogger("component:TokenSideBarContent");
 import getUserRoles, { UserRole } from "../../utils/UserRolesUtil";
 import { useSelector } from "react-redux";
 import { NETWORK_NAME } from "../../config";
+const { trace, error } = getLogger("component:TokenSideBarContent");
```

Comment:
this change required ?

Reply:
`npm run storybook` was complaining about its order.